cypress/integration/checkLists.ts

```diff
@@ -0,0 +1,31 @@
+describe('On check-lists example', () => {
```

Comment:
Could you and @samarsault put all these tests in something like `integration/examples/`? There will be non-example-based cypress tests in the future probably, and this groups them nicely.

```suggestion
describe('Check-lists example', () => {
```